tests/test_livepatch.py

```diff
@@ -72,46 +73,47 @@ def test_enable_livepatch_apt_get_options(self):
 
     def test_enable_livepatch_installs_snap(self):
         """enable-livepatch installs the livepatch snap if needed."""
-        self.setup_livepatch(installed=False)
+        self.setup_livepatch(enabled=False)
```

Comment:
Can this setup_livepatch() be dropped, since the default for enabled is False already?

Reply:
Well, that depends on the comment above, since the default is `None` at the moment